cms/djangoapps/contentstore/migrations/0002_add_assets_page_flag.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
So django handles the python-MySQL interaction automagically, yeah? Migrations are how you tell django "my data model has changed, please update the SQL tables to match".

These docs are pretty thorough: https://docs.djangoproject.com/en/1.11/topics/migrations/